ml/src/test/scala/ai/h2o/sparkling/ml/algos/H2OGridSearchTestSuite.scala

```diff
@@ -155,6 +154,7 @@ class H2OGridSearchTestSuite extends FunSuite with Matchers with SharedH2OTestCo
       .setLabelCol("AGE")
       .setHyperParameters(hyperParams)
       .setAlgo(algo)
+      .setStrategy()
```

Comment:
Argument missing to setStrategy

Reply:
oh, good catch, this file shouldn't have been committed